app/views/pages/developer/literature.nunjucks

```diff
@@ -0,0 +1,91 @@
+{% extends ".nunjucks ./../shared/layout/html/html.nunjucks" %}
+{% import "./developer.macros.nunjucks" as api %}
+
+{% macro trow(url, resp, httpMethod="GET", respLink, paging=undefined, params=[]) %}
+    {$ api.trow(url="/literature"+url, httpMethod=httpMethod, resp=resp, respLink=respLink, params=params, paging=paging, authRequired=auth, description=caller()) $}
+{% endmacro %}
+
+
+{% block page %}
+
+    <article dir="ltr" class="wrapper-horizontal-stripes page--developer">
+        <section class="horizontal-stripe article-header white-background">
+            <div class="container">
+                <div class="row">
+                    <header class="col-xs-12 text-center">
+                        <nav class="article-header__category article-header__category--deep">
+                            <span class="article-header__category__upper">Developer</span>
+                            <span class="article-header__category__lower">API Docs</span>
+                        </nav>
+                        <h1>Literature API</h1>
+                        <div class="article-header__intro">
+                            <p>{$ apiBase $}</p>
+                        </div>
+                    </header>
+                </div>
+            </div>
+        </section>
+
+        {$ api.apiDocTabs('literature', __) $}
+
+        <section id="overview" class="horizontal-stripe light-background">
+            <div class="container">
+                <p>
+                    This API is used to retrieve literature.
```

Comment:
Suggest change this to:

> This API enables you to search for literature indexed by GBIF, including peer-reviewed papers, citing GBIF datasets and downloads. It powers the <a href="https://www.gbif.org/resource/search?contentType=literature">Literature search</a> on GBIF.org.

Reply:
Thank you, I changed that. I deployed an updated version